src/Blog/NewsBundle/Repository/CategoryRepository.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Blog\NewsBundle\Repository;
+
+use Doctrine\ORM\EntityRepository;
+
+class CategoryRepository extends EntityRepository
+{
+    public function getCategories()
```

Comment:
no need of wrapping parent method findAll (Doctrine\ORM\EntityRepository) in your own method